src/git-duet/git-duet-rebase/main.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"git-duet/internal/cmd"
+	"git-duet/internal/cmdrunner"
+)
+
+func main() {
+	err := cmdrunner.Execute(
+		cmd.New("rebase"))
```

Comment:
Interesting - it looks like `git rebase` uses the `GIT_COMMITTER_NAME` and `GIT_COMMITTER_EMAIL`, and there are a couple of scenarios in which a rebase may change the committer.  Would this work the way you would expect?

For example, let's say @jszwedko and I are pairing.  He's driving at the moment, so he sets the info like so:
 `git duet js rc`